### ISSUE TITLE
_grid-basic.scss needs an import for variables/_var

### ISSUE THREAD

**Issue by Christoph-Wagner** [lodog23/Taffy-CSS-Framework#1]
Using webpack to load taffy.scss I get the following error:

    ERROR  Failed to compile with 1 errors
    error  in ./src/assets/styles/taffy-0.6.0/scss/grid/_grid-basic.scss
    
    Module build failed: 
    undefined
              ^
          Undefined variable: "$grid-gutter".
          in {…path…}\src\assets\styles\taffy-0.6.0\scss\grid\_grid-basic.scss (line 14, column 12)

Adding `@import "../variables/_var";` to the top of _grid-basic.scss fixes it.

**Comment by lodog23** [comment]
Were you only importing the _grid-basic.scss file and not the _var.scss file, before you added it to the top of _grid-basic.scss?

If that is the case the reasoning is in _var.scss I set it so the user can change the size of the grid gutter to what they want.

If that is not the case and you were importing both the _grid-basic.scss and _var.scss files let me know so I can resolve the issue!

**Comment by Christoph-Wagner** [comment]
I imported only the main file, taffy.scss

**Comment by lodog23** [comment]
Hmm that's odd. You had the other folders in your project too correct? 

Was that the only variable that was giving you issues?

**Comment by Christoph-Wagner** [comment]
Yeah, everything is there and besides that one file everything else works without an error.

I'm using node-sass 4.5.3 via webpack 3.0.0
